quarkus-build-caching-extension/README.md

```diff
@@ -105,6 +109,56 @@ The configuration file location can either be defined:
 - as a maven property:
 `<gradle.quarkus.extension.config.file>.quarkus/extension-local.properties</gradle.quarkus.extension.config.file>`
 
+## Implementation details
+
+The instructions required to make the Quarkus `build` goal cacheable are isolated to the [QuarkusBuildCache](./src/main/java/com/gradle/QuarkusBuildCache.java) class.
```

Comment:
by instructions here do you mean instructions for the consumer of the caching utility? Or do you mean the code to make the Quarkus build goal cacheable?

Reply:
I meant the code, let me rephrase to clarify with
> The logic to make the Quarkus `build` goal cacheable is isolated to the [QuarkusBuildCache](./src/main/java/com/gradle/QuarkusBuildCache.java) class.

Reply:
sounds good